extra-dev/packages/coq-fcsl-pcm/coq-fcsl-pcm.dev/opam

```diff
@@ -10,8 +10,7 @@ license: "Apache-2.0"
 build: [ make "-j%{jobs}%" ]
 install: [ make "install" ]
 depends: [
-  "ocaml"
-  "coq" {(>= "8.10" & < "8.12~") | (= "dev")}
+  "coq" {(>= "8.10" & < "8.13~") | (= "dev")}
   "coq-mathcomp-ssreflect" {(= "dev")}
```

Comment:
```suggestion
  "coq-mathcomp-ssreflect" {(>= "1.10.0" & < "1.12~") | (= "dev")}
```
This is what the repo has, is there any reason to diverge from the repo here?

Reply:
At some point I used some features that were available in the dev version only. Now it makes sense to add this back.

Reply:
Thanks!